### PR TITLE
Staging release notes for Nov 13 2019

### PR DESCRIPTION
**Pull request by rhiananthony**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
 
+# Staging 2019/11/07 Release Notes
+## Ingest
+### Version(s):
+#### Core:
+* Deploying and Testing Mongo Migrations bump our used mongo version
+* Feature compatibility 3.4
+* Server 3.6 & Feature compatibility 3.6
+* Deployment failed the DCP staging tests 
+* 2019/11/12 Resolved by restarting the ingest core
```
Comment (MightyAx) [inline RELEASE_NOTES.md]: These two can be deleted

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # Release Notes
 
+# Staging 2019/11/07 Release Notes
```
Comment (MightyAx) [inline RELEASE_NOTES.md]: 2019/11/13?

File: RELEASE_NOTES.md (hunk 1 above)
Comment (rhiananthony) [reply RELEASE_NOTES.md]: Thanks!

File: RELEASE_NOTES.md (hunk 2 above)
Comment (rhiananthony) [reply RELEASE_NOTES.md]: Doh - copy error

**Pull request merged**